chatmaild/src/chatmaild/ini/chatmail.ini.f

```diff
@@ -20,6 +20,9 @@
 # days after which mails are unconditionally deleted
 delete_mails_after = 20
 
+# days after which accounts are deleted if there were no login
+delete_accounts_after = 30
```

Comment:
maybe 21 could be the default? This is exactly the GDPR limit iirc. Then we wouldn't have to do any manual action if users want to delete their account, we could just tell them to delete it in their app.

Reply:
In that case,let's do 20, like with the mails, I'd say

Reply:
the GDPR limit is rather 30 days IIRC, so 25 or so should be fine.

Reply:
set it to 25